basic/02a_feedback_foreground.cpp

```diff
@@ -3,14 +3,17 @@
 #include <thread>
 #include "lookup.hpp"
 #include "group_feedback.hpp"
+#include "matplotlibcpp.h"
 
 using namespace hebi;
 
+namespace plt = matplotlibcpp;
+
 int main()
 {
   // Get a group
   Lookup lookup;
-  auto group = lookup.getGroupFromNames({ "Test Family" }, { "Test Actuator" });
+  auto group = lookup.getGroupFromNames({ "HEBI" }, { "X-00147" });
```

Comment:
Make sure to change this back. I know it's annoying to have to edit `Test Family`, etc to get it to work on your local setup, but I think it's good to have us stay consistent in the examples with `Test Family`, `Test Actuator`, etc...